shell/common/gin_converters/content_converter.cc

```diff
@@ -259,6 +259,8 @@ v8::Local<v8::Value> Converter<blink::PermissionType>::ToV8(
       return StringToV8(isolate, "smart-card");
     case blink::PermissionType::WEB_PRINTING:
       return StringToV8(isolate, "web-printing");
+    case blink::PermissionType::SPEAKER_SELECTION:
+      return StringToV8(isolate, "speaker-selection");
```

Comment:
This needs to be added to the list in the docs too

Reply:
@MarshallOfSound its a placeholder that currently doesn't do anything: 5278243: Add placeholder of speaker-selection permission. | https://chromium-review.googlesource.com/c/chromium/src/+/5278243

Reply:
@MarshallOfSound also what docs?